Source/GUI/MainWindow.cpp

```diff
@@ -84,6 +84,7 @@ void MainWindow::initialiseWidgets()
   m_btnSetSeedManually->setEnabled(false);
 
   m_lblCurrentSeed = new QLabel("  ????  ");
+  m_lblCurrentSeed->setTextInteractionFlags(NULL);
```

Comment:
Don't use `NULL` as it's an old C macro that shouldn't be used anymore. Qt also has an enum to better say what you are doing: `Qt::NoTextInteraction` as outlined here https://doc.qt.io/qt-5/qt.html#TextInteractionFlag-enum . There's also deprecation warnings due to this.